### PR TITLE
feat(api-client): hotkeys modifiers

### PR DESCRIPTION
**Pull request by antlio**
this pr is a follow up on #2814 and aims to:
- enable multiple hotkey modifier  usage (e.g. meta + alt + key)
- add topnav hotkeys
- add addressbar focus hotkey

### CODE REVIEW

File: packages/api-client/src/components/AddressBar/AddressBar.vue [hunk 1]
```diff
@@ -109,6 +115,20 @@ const updateExampleUrlHandler = (url: string) => {
...
+    addressBarRef.value?.focus()
+  }
+}
+
+onMounted(() => {
+  hotKeyBus.on(handleHotKey)
+})
+onBeforeUnmount(() => {
+  hotKeyBus.off(handleHotKey)
+})
```
Comment (amritk) [inline packages/api-client/src/components/AddressBar/AddressBar.vue:131]:
```suggestion
onMounted(() => hotKeyBus.on(handleHotKey))
onBeforeUnmount(() => hotKeyBus.off(handleHotKey))
```
when you only have one, it looks a bit cleaner 😁

**Review by amritk — APPROVED** [review]
Code looks good to me, untested but well done!

**Review by marclave — APPROVED** [review]
AWESOME!

File: packages/api-client/src/components/AddressBar/AddressBar.vue (hunk 1 above)
Comment (antlio) [reply packages/api-client/src/components/AddressBar/AddressBar.vue:131]: 🍬 absolutely lets go!

**Pull request merged**